### PR TITLE
DockerfileParser: add 'value' key to structure property and make get_* functions use it

### PR DESCRIPTION
**Pull request by jpopelka**
@twaugh Please review

### CODE REVIEW

File: dock/util.py [hunk 1]
```diff
@@ -176,19 +176,27 @@ def structure(self):
...
-             "content": "From fedora\n"},
+             "content": "From fedora\n",
+             "value": "fedora"},
 
             {"instruction": "CMD",
              "startline": 1,
              "endline": 2,
-             "content": "CMD yum -y update && \\\n    yum clean all\n"}
+             "content": "CMD yum -y update && \\\n    yum clean all\n",
+             "value": "yum -y update && yum clean all"}
```
Comment (twaugh) [inline dock/util.py]: What would the value be expected to be for `"CMD yum -y update &&     \\\n  yum clean all"`? Currently leading whitespace is stripped while whitespace up to `\` is preserved. Is that what docker does? What about `"RUN ls&&\\\nusb\n"`?

**Comment by twaugh** [comment]
Oh, right, there's a `test_util.py`, hadn't thought of looking for it. I've got a test-case for `.structure` to add to that after this PR.

**Comment by twaugh** [comment]
Looks really good. I don't know enough about how docker handles whitespace...

**Comment by twaugh** [comment]
OK, the answer is that docker strips whitespace between the instruction and the first character of the value, and after the last character of the value, but not between continuation lines.

So I think the `lstrip()` should only happen for the first line of a multiline set, and `rstrip()` should only happen for the last.

**Comment by twaugh** [comment]
FWIW, here's the test-case I've written: d88e1e21.

**Comment by twaugh** [comment]
Actually, better one: 92afe022.

**Comment by twaugh** [comment]
Looks ready for merge. (Sorry about the typo in the test...)

**Comment by jpopelka** [comment]
@twaugh could you review also the last commit (I haven't touched the others). It fixes #181.

I tested that docker parse it the same via putting into Dockerfile:
LABEL label11 11
LABEL label12 1 2
LABEL "label13" 1 3
LABEL label14 "1" 4
LABEL label15 1 '5'
then `docker build` & `docker inspect` results in:
        "Labels": {
            "label11": "11",
            "label12": "1 2",
            "label13": "1 3",
            "label14": "1 4",
            "label15": "1 5"
        }

**Comment by twaugh** [comment]
:+1:

**Pull request merged**